extensions/src/ACRE2Core/ext_remoteStopSpeaking.h

```diff
@@ -21,11 +21,11 @@ RPC_FUNCTION(ext_remoteStopSpeaking) {
             this->getSelf()->getCurveScale()
         ) */
 
-    ACRE_ID id = (ACRE_ID)vMessage->getParameterAsInt(0);
+    ACRE_ID id = (ACRE_ID) vMessage->getParameterAsInt(0);
```

Comment:
static_cast and auto?
Is the cast even necessary? Can't it implictly convert on =?